libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -899,6 +900,28 @@ const ParentalLeaveTemplate: ApplicationTemplate<
 
         return context
       }),
+      removePeriodsOrAllowanceOnSpouseRejection: assign((context) => {
+        const { application } = context
+
+        const answers = getApplicationAnswers(application.answers)
+
+        if (answers.requestDays > 0) {
```

Comment:
Add reset on `giveRights` too. Sometime user gives right to otherParent `"giveRights": { "giveDays": "0", "isGivingRights": "no" }`